docs/src/examples/components/Dropdown/Types/DropdownExampleMultipleSelection.shorthand.steps.ts

```diff
@@ -0,0 +1,32 @@
+import { Dropdown, DropdownSelectedItem } from '@stardust-ui/react'
+
+const selectors = {
+  triggerButton: `.${Dropdown.slotClassNames.triggerButton}`,
+  dropdownItem: (itemIndex = 1) =>
+    `.${Dropdown.slotClassNames.itemsList} li:nth-child(${itemIndex})`,
+  dropdownSelectedItemRemoveIcon: (itemIndex = 1) =>
+    `.${Dropdown.slotClassNames.selectedItems} span:nth-child(${itemIndex}) .${
+      DropdownSelectedItem.slotClassNames.removeIcon
+    }`,
+}
+
+const steps = [
+  steps =>
+    steps
+      .click(selectors.triggerButton)
+      .click(selectors.dropdownItem(3))
+      .click(selectors.triggerButton)
+      .click(selectors.dropdownItem(2))
+      .click(selectors.triggerButton)
+      .snapshot('Selects 2 items and opens list'),
+  steps =>
+    steps
+      .click(selectors.dropdownSelectedItemRemoveIcon())
+      .click(selectors.triggerButton)
+      .click(selectors.dropdownSelectedItemRemoveIcon())
+      .click(selectors.triggerButton)
+      .snapshot('Removes the 2 items and sees the initial list')
+      .click(selectors.triggerButton), // close the list, no need for new screenshot
```

Comment:
but why did you close it since you finished the test already?

Reply:
if someone adds a new test I wanted to leave the Dropdown in an initial state; suggestions? Is that necessary?

Reply:
removed that call